templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<h1>Delete Manufacturer</h1>
+
+<p>Are you sure you want to delete the manufacturer: {{ manufacturer }}?</p>
+<p>All Cars of this manufacturer will also be deleted!</p>
+
+<form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">

```

Comment:
What if a user changed his mind, and doesn't want to delete it (yes he can press any link), but add a go-back button